spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -958,9 +1289,6 @@ def write_data_spec(
         gen_data = self._write_synaptic_matrix_and_master_population_table(
             spec, post_slices, post_slice_idx, machine_vertex,
             post_vertex_slice, all_syn_block_sz, weight_scales,
-            POPULATION_BASED_REGIONS.POPULATION_TABLE.value,
-            POPULATION_BASED_REGIONS.SYNAPTIC_MATRIX.value,
-            POPULATION_BASED_REGIONS.DIRECT_MATRIX.value,
             routing_info, graph_mapper, machine_graph, machine_time_step)
```

Comment:
NO NO NO NO NO! we have to be able to support other vertices which are not APV which utilise a Synaptic manager. by forcing the regions to be that of APV is clunky. retrack to original format.